pkg/kyverno/test/test_command.go

```diff
@@ -161,7 +161,7 @@ func Command() *cobra.Command {
 			return nil
 		},
 	}
-	cmd.Flags().StringVarP(&fileName, "file-name", "f", "test.yaml", "test filename")
+	cmd.Flags().StringVarP(&fileName, "file-name", "f", "kyverno-test.yaml", "test filename")
```

Comment:
`test.yaml` won't be supported in 1.6? How do we guarantee backward compatibility?

Reply:
test.yaml are supported in 1.6. Both file name supported in 1.6. In later release will deprecated test.yaml. I have handle in code test.yaml file name.

Reply:
Right! `kyverno-test.yaml` is the default value here.